# name: resource.error 
exception resource.error